Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -113,7 +112,13 @@ void MolDraw2D::doContinuousHighlighting(
   PRECONDITION(activeMolIdx_ >= 0, "bad active mol");
 
   int orig_lw = lineWidth();
+  bool orig_lws = drawOptions().scaleBondWidth;
+  drawOptions().scaleBondWidth = true;
```

Comment:
same comment here that I made below: it seems to me that it's confusing/counter-intuitive to scale the highlighting if we aren't scaling the bond widths. Is there a particular example that led to this?

Reply:
After off-line discussion, drawOptions().scaleHighlightBondWidth has been added.